tests/integration/targets/lookup_sops/tasks/main.yml

```diff
@@ -116,3 +116,15 @@
           "a: b\nc: d"
         test_str_binary_data: This is binary data.
         test_str_abcd: a is b, and c is d
+
+    - name: Test fake sops binary
+      set_fact:
+        fake_sops_output: "{{ lookup('community.sops.sops', 'simple.sops.yaml', sops_binary=role_path ~ '/files/fake-sops.sh', enable_local_keyservice=True, aws_access_key_id='xxx') }}"
```

Comment:
is `~` used as string concatenation here?

Reply:
Yes, that's Jinja2 standard syntax for string concatenation (https://jinja.palletsprojects.com/en/2.11.x/templates/#other-operators).